Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/TranslationHandles.cs

```diff
@@ -0,0 +1,338 @@
+            GetVisual(handle).transform.localScale = new Vector3(invScale, invScale, invScale);
+            Vector3 colliderSizeScaled = visualSize * invScale;
+            if (config.TranslationHandlePrefabColliderType == HandlePrefabCollider.Box)
+            {
+                BoxCollider collider = handle.gameObject.GetComponent<BoxCollider>();
+                collider.size = colliderSizeScaled;
+                collider.size += BaseConfig.ColliderPadding;
+            }
+            else
+            {
+                SphereCollider collider = handle.gameObject.GetComponent<SphereCollider>();
+                collider.radius = colliderSizeScaled.x * 0.5f;
+                collider.radius += VisualUtils.GetMaxComponent(config.ColliderPadding);
+            }
+        }
+
+
+        protected Bounds CreateVisual(int handleIndex, GameObject parent)
+        {
+            GameObject midpointVisual;
+            GameObject prefabType = config.HandlePrefab;
+            if (prefabType != null)
+            {
+                midpointVisual = Object.Instantiate(prefabType);
+            }
+            else
+            {
+                midpointVisual = GameObject.CreatePrimitive(PrimitiveType.Sphere);
+                Object.Destroy(midpointVisual.GetComponent<SphereCollider>());
+            }
+
+            // Even handle indices point in the positive direction along their axis,
+            // and odd handle indices point in the negative direction;
+            var directionSign = handleIndex % 2 == 0 ? 1.0f : -1.0f;
+
+            // Align handle with its edge assuming that the prefab is initially aligned with the up direction 
+            if (faceAxes[handleIndex] == CardinalAxisType.X)
+            {
+                Quaternion realignment = Quaternion.FromToRotation(Vector3.forward, directionSign * Vector3.right);
+                midpointVisual.transform.localRotation = realignment * midpointVisual.transform.localRotation;
+            }
+            else if (faceAxes[handleIndex] == CardinalAxisType.Z)
+            {
+                Quaternion realignment = Quaternion.FromToRotation(Vector3.forward, directionSign * Vector3.forward);
+                midpointVisual.transform.localRotation = realignment * midpointVisual.transform.localRotation;
+            } else if (faceAxes[handleIndex] == CardinalAxisType.Y)
+            {
+                Quaternion realignment = Quaternion.FromToRotation(Vector3.forward, directionSign * Vector3.up);
+                midpointVisual.transform.localRotation = realignment * midpointVisual.transform.localRotation;
+            }
+
+            Bounds midpointBounds = VisualUtils.GetMaxBounds(midpointVisual);
+            float maxDim = VisualUtils.GetMaxComponent(midpointBounds.size);
+            float invScale = maxDim == 0.0f ? 0.0f : config.HandleSize / maxDim;
+
+            midpointVisual.name = "visuals";
```

Comment:
Turn "visuals" into a const string at the head of this class, and replace the references to it with that const string variable.

There are a few places that use this explicit string and this makes it somewhat fragile (i.e. super easy to break by changing the string in one location but not the other)

Reply:
Deferring to @thalbern here. The other handles use this pattern.

Reply:
replaced by static string in base handles